util/osutil/path_windows.go

```diff
@@ -1,10 +1,8 @@
-package build
+package osutil
 
-import "golang.org/x/sys/windows"
```

Comment:
Looks missing

Reply:
Good catch. I'll have to watch for that in the future. I didn't realize my IDE wasn't running `goimports` on files that didn't match my build constraints.